### ISSUE TITLE
Easy accessibility fixes

### ISSUE THREAD

**Issue by arildm** [spraakbanken/mink-frontend#176]
The Wave browser extension reports some issues that are easy to fix:

- Some images are missing alt text
- Some buttons and tables are missing text
- The language selector is missing a label
- There is no h1 heading

Another issue is the contrast on some links (orange on white or light gray) and buttons (white on blue). We can look into it but it doesn't seem as easy to fix.

**Comment by arildm** [comment]
Fixed in these recent commits:

* 14f377f - fix(a11y): h1 on Home (Tue Nov 19 13:45:08 2024 +0100) <Arild Matsson>
* ...
* f38c3c9 - style(a11y): links underlined, not orange (Tue Nov 19 10:18:33 2024 +0100) <Arild Matsson>
* 25061b4 - style(a11y): brighter progress bar (Tue Nov 19 10:17:28 2024 +0100) <Arild Matsson>
* ec8dbda - fix(a11y): warning button dark (Tue Nov 19 10:07:46 2024 +0100) <Arild Matsson>
* 97d8dd1 - fix(a11y): dark mode contrast (Mon Nov 18 18:44:51 2024 +0100) <Arild Matsson>
* a7b4d21 - fix(a11y): darker orange (Mon Nov 18 18:23:06 2024 +0100) <Arild Matsson>
* a23b6a9 - fix(a11y): more contrast (Mon Nov 18 18:18:24 2024 +0100) <Arild Matsson>
* 628ad9a - fix(a11y): avoid redundant adjacent links (Mon Nov 18 18:18:24 2024 +0100) <Arild Matsson>
* 28425a3 - fix(a11y): empty th, button (Mon Nov 18 18:18:20 2024 +0100) <Arild Matsson>
* 9de8c35 - fix(a11y): better tab key navigation (Mon Nov 18 17:24:05 2024 +0100) <Arild Matsson>
* 0d2c6d5 - fix(a11y): orphaned label for identifier (Mon Nov 18 17:05:31 2024 +0100) <Arild Matsson>
* e56638d - fix(a11y): LocaleSwitcher hidden label (Mon Nov 18 16:58:43 2024 +0100) <Arild Matsson>
* d7973ca - fix(a11y): img alt (Mon Nov 18 16:48:09 2024 +0100) <Arild Matsson>
